style.css

```diff
@@ -545,15 +554,78 @@ div#frams {
   padding: 20px;
   align-items: center;
   position: absolute;
-  bottom: 100px;
-  left: 30px;
-  right: 30px;
+  bottom: 30px;
+}
+
+.inputs {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  border: #f4f5f7;
+}
+
+#nam {
+  width: 327px;
+  height: 48px;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+  border: #fff5e1;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.contn {
```

Comment:
- **[Optional]** I think it would be better to make your mobile version responsive until the desktop screen size. This will allow for a consistent user experience across different devices. 👍 
  
![image](https://user-images.githubusercontent.com/102757126/213126161-cf5cce1d-8d59-4041-a56a-4c829ddb1134.png)